src/Integration.Vsix.UnitTests/CFamily/PreCompiledHeaders/PchFilesDeleterTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO.Abstractions.TestingHelpers;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarLint.VisualStudio.Integration.Vsix.CFamily;
+
+namespace SonarLint.VisualStudio.Integration.UnitTests.CFamily.PreCompiledHeaders
+{
+    [TestClass]
+    public class PchFilesDeleterTests
+    {
+        private MockFileSystem fileSystemMock;
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            fileSystemMock = new MockFileSystem();
```

Comment:
I'm using a mock file system since I want to check the searchFilter, and it's supported by the mock.